packages/feel-input-component/src/FeelConfigs.ts

```diff
@@ -25,11 +26,15 @@ export const feelTheme = (): Monaco.editor.IStandaloneThemeData => {
     base: "vs",
     inherit: true,
     rules: [
-      { token: "feel-keyword", foreground: "26268C", fontStyle: "bold" },
-      { token: "feel-numeric", foreground: "1750EB" },
-      { token: "feel-boolean", foreground: "26268D", fontStyle: "bold" },
-      { token: "feel-string", foreground: "067D17" },
-      { token: "feel-function", foreground: "00627A" },
+      { token: "feel-keyword", foreground: "#26268C", fontStyle: "bold" },
+      { token: "feel-numeric", foreground: "#1750EB" },
+      { token: "feel-boolean", foreground: "#26268D", fontStyle: "bold" },
+      { token: "feel-string", foreground: "#067D17" },
+      { token: "feel-function", foreground: "#00627A" },
+      { token: "input-node", foreground: "#917632", fontStyle: "underline" },
+      { token: "bkm-node", foreground: "#917632", fontStyle: "underline italic" },
+      { token: "invalid-variable", foreground: "#ff0000", fontStyle: "underline bold" },
+      { token: "parameter", foreground: "#036e9b", fontStyle: "italic" },
```

Comment:
The tokens names can be improved:

```suggestion
      { token: "input-data-variable", foreground: "#917632", fontStyle: "underline" },
      { token: "bkm-variable", foreground: "#917632", fontStyle: "underline italic" },
      { token: "unknown-variable", foreground: "#ff0000", fontStyle: "underline bold" },
      { token: "function-parameter-variable", foreground: "#036e9b", fontStyle: "italic" },
```

`input-node` --> "Node" is a diagram element. We're not dealing with Diagrams here. Also, the full name of "inputs" on DMN is "InputData". I would call this token "input-data-variable".
`bkm-variable` --> Same as above.
`invalid-variable`. --> I don't think "invalid" is fair. This token is used for when a variable is detected, but it's unknown.
`parameter` --> Too generic. `function-parameter-variable` improves it a little bit IMHO.